docs/contributing/index.md

```diff
@@ -58,7 +58,7 @@ project. Please read this section carefully if you are interested in contributin
 	`npm run lint` with no warnings.
 	*   Please squash pull requests down to a single commit to simplify review and keep history clean.
 	*   Ensure all new features have unit tests that pass and any bug fixes include tests that
-	fail without the fix applied
+	fail without the fix applied. If you want to test for ui or screenshot, refer the [UI-Test](./ui-testing.md)/[Screenshot](./screenshot-testing.md) page.
```

Comment:
*   Ensure all new features have unit tests, [ui tests](./ui-testing.md), and /[screenshot tests](./screenshot-testing.md) that pass and any bug fixes include tests that fail without the fix applied. 

I think we should add ui/screesnhot test next to unit test.